### DISCUSSION TITLE
Custom field type "Label", usable with Drafts

### DISCUSSION THREAD

**Discussion by gabriel-r** [community/community#46490] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

We use a single Project to manage Issues across multiple Repositories. 

It would be very useful (necessary) to have a custom field of type Label, to select and share them across all repos in the project. 

We would use it to:
* assign Labels to Drafts
* group issues affecting a Product, or Customer
* group issues under Epics 
* harmonize key labels across repos

This is related to https://github.com/github/roadmap/issues/276

**Comment by gabriel-r** [comment]
PS: Yeah, so the OG ticket actually mentions "Cross repository milestones _and labels_"

Maybe one is easier than the other? Either Milestones or Labels at project or org level would be super useful.

**Comment by Andrej730** [comment]
Assigning labels to drafts would be super useful.
